Week1/homework/create-database.js

```diff
@@ -0,0 +1,53 @@
+const util = require('util');
+const mysql = require('mysql');
+const countries = require('./countries.js');
+const cities = require('./cities.js');
+
+const connection = mysql.createConnection({
+  multipleStatements: true,
+  host: 'localhost',
+  user: 'hyfuser',
+  password: '7566',
+  database: 'userdb'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+  const CREATE_DATABASE_AND_LOGIN = `
+    CREATE DATABASE IF NOT EXISTS world; USE world`;
+
+  const CREATE_COUNTRY_TABLE = `
+    CREATE TABLE IF NOT EXISTS country (
+      Name TEXT,
+      Population INT,
+      Surface_Area_km2 INT,
+      Continent TEXT
+    );`;
+
+  const CREATE_CITY_TABLE = `
+    CREATE TABLE IF NOT EXISTS city (
```

Comment:
This schema is perhaps enough for the homework, but a better practice is to also give each row an ID. We will talk about in next lessons, pay attention to "key"s.

Reply:
IDs are added.